teachers_digital_platform/tests/models/test_activity_page.py

```diff
@@ -0,0 +1,91 @@
+from django.test import TestCase
+
+from wagtail.wagtaildocs.models import Document
+
+from model_mommy import mommy
+from teachers_digital_platform.models import (
+    ActivityAgeRange, ActivityBloomsTaxonomyLevel, ActivityBuildingBlock,
+    ActivityCouncilForEconEd, ActivityDuration, ActivityGradeLevel,
+    ActivityJumpStartCoalition, ActivityPage, ActivitySchoolSubject,
+    ActivityTeachingStrategy, ActivityTopic, ActivityType
+)
+
+
+class TestActivityPage(TestCase):
+    fixtures = ['tdp_initial_data']
+
+    def setUp(self):
+        super(TestActivityPage, self).setUp()
+
+    def test_get_subtopic_ids_returns_correct_subtopics(self):
+        # Arrange
+        activity_page = self.create_activity_detail_page('Test', 'test')
+        # Act
+        actual_subtopic_ids = activity_page.get_subtopic_ids()
+        # Assert
+        self.assertTrue(6 not in actual_subtopic_ids)
+        self.assertTrue(7 in actual_subtopic_ids)
+
+    def test_get_subtopic_ids_works_with_no_topics(self):
+        # Arrange
+        activity_page = self.create_activity_detail_page('Test', 'test', topic_list=[])  # noqa: E501
+        # Act
+        actual_subtopic_ids = activity_page.get_subtopic_ids()
+        # Assert
+        self.assertTrue(isinstance(actual_subtopic_ids, (list, set)))
```

Comment:
This and the other instance tests could use [`assertIsInstance'](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsInstance) (and we know the subtopics will always be a set and the grade levels will always be a list):

```suggestion
        self.assertIsInstance(actual_subtopic_ids, set)
```